README.md

```diff
@@ -46,27 +46,85 @@ It correctly bundles React in production mode and optimizes the build for the be
 ## Workflow
 
 - **Create new ```feature-branch``` from ```develop``` branch that describes your work.**
+
+  ```bash
+    git checkout -b project-setup
+  ```
+
 Example:
-![alt text](https://res.cloudinary.com/elbon/image/upload/v1566291009/branch-naming.png)
+![Create a New Branch](https://res.cloudinary.com/elbon/image/upload/v1566291009/branch-naming.png)
 
 - **Adding a New Feature**
   - Create Component
+
+    ```js
+      const Home = () => <h1>Welcome to BravoFE</h1>;
```

Comment:
Can you add the path to the folder where you need to create this file. so, something like on components - 1. componentfolderName 2. Component.js. 

 what should the file be named. Put the folder structure layout before these steps.